openpype/settings/entities/schemas/projects_schema/schema_project_photoshop.json

```diff
@@ -131,6 +131,28 @@
                         }
                     ]
                 },
+                {
+                    "type": "dict",
+                    "collapsible": true,
+                    "key": "CollectVersion",
+                    "label": "Collect Version",
+                    "children": [
+                        {
+                            "type": "boolean",
+                            "key": "optional",
+                            "label": "Optional"
+                        },
+                        {
+                            "type": "label",
+                            "label": "Synchronize version for image and review instances by workfile version."
+                        },
+                        {
```

Comment:
I would replace `sync_workfile_version` with `enabled`. It doesn't make sense to show the plugin if it's functionality is disabled.

Reply:
Just realized it's collector... You can't have optional collector (maybe in new publisher?).